pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -150,6 +150,12 @@ type ElasticsearchSpec struct {
 
 	// +optional
 	RedundancyPolicy elasticsearch.RedundancyPolicyType `json:"redundancyPolicy"`
+
+	// The resource requirements for ElasticsearchProxy
+	//
+	// +nullable
+	// +optional
+	ProxyResources *v1.ResourceRequirements `json:"proxyResources"`
```

Comment:
To keep following the pattern (e.g. See `KibanaSpec.ProxySpec`), splitting to a separate `ProxySpec` struct is more appropriate. Grouping proxy configuration in one new struct enables backward compatible additions. (cc @ewolinetz )

Reply:
updated